apps/client/src/domains/routing/assets/time.svgr.tsx

```
@@ -0,0 +1,22 @@
+const SvgTime = (props) => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width={props.size}
```

ESLint

[error] 'size' is missing in props validation

(react/prop-types)